events/syscall/read_environment_variable_from_proc_files.go

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os/exec"
+    
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(
+    ReadEnvironmentVariableFromProcFiles,
+    events.WithDisabled(), // this rules is not included in falco_rules.yaml (stable rules), so disable the action
+)
+
+func ReadEnvironmentVariableFromProcFiles(h events.Helper) error {
+    if h.InContainer() {
+        cmd := exec.Command("cat", "/proc/*/environ")
```

Comment:
Can we use just `/proc/self/environ` instead?

Reply:
Yes, @FedeDP, I changed it, and the rule was also triggered.